source/main/utils/HandleDiskSpace.js

```diff
@@ -0,0 +1,71 @@
+// @flow
```

Comment:
Since this is not a class i would rather use lower case letter for the first char of the file name (`handleDiskSpace` or maybe even `disk-space.js`)